HISTORY.md

```diff
@@ -1,5 +1,6 @@
-### mlpack ?.?.?
+### mlpack 3.3.3
```

Comment:
```suggestion
### mlpack ?.?.?
```
We note like that since it's not released, and it may, for example be 3.4.0; we don't know yet.

Reply:
Oh, never knew this, thanks!